tests/server/routes.js

```diff
@@ -77,6 +77,7 @@ var routes = {
   '/sms/sent': { statusCode: 200 },
   '/sms/why': { statusCode: 200 },
   // the following have a version prefix
+  '/v1/authorization': { statusCode: 200 },
```

Comment:
IIUC, the existing routes that have a "/v1" prefix have it for legacy reasons.  I don't think we need to add support for it on this route, unless we expect any existing links to "https://accounts.firefox.com/v1/authorization" to be live in the wild.